geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -157,6 +159,28 @@ pub struct ReplicaTransactionInfoV2<'a> {
     pub index: usize,
 }
 
+/// Information about a transaction, including index in block and post accounts states data
+#[derive(Clone, Debug)]
+#[repr(C)]
+pub struct ReplicaTransactionInfoV3<'a> {
+    /// The first signature of the transaction, used for identifying the transaction.
+    pub signature: &'a Signature,
+
+    /// Indicates if the transaction is a simple vote transaction.
+    pub is_vote: bool,
+
+    /// The sanitized transaction.
+    pub transaction: &'a SanitizedTransaction,
+
+    /// Metadata of the transaction status.
+    pub transaction_status_meta: &'a TransactionStatusMeta,
+
+    /// The transaction's index in the block
+    pub index: usize,
+    /// States of accounts that was in transaction
+    pub post_accounts_states: Vec<(Pubkey, AccountSharedData)>,
```

Comment:
*style*

All the other fields are separate by an empty line.
Seems reasonable to follow the same style for the new fields as well.

```suggestion
    /// The transaction's index in the block
    pub index: usize,

    /// States of accounts that were involved in the transaction
    pub post_accounts_states: Vec<(Pubkey, AccountSharedData)>,
```